### PR TITLE
Upgrade f* packages

### PR DESCRIPTION
**Pull request by Zopolis4**
Most of them, anyways.

### Run the following to get this pull request's changes locally for testing.
```bash
CREW_REPO=https://github.com/Zopolis4/chromebrew.git CREW_BRANCH=1muck crew update
```

### CODE REVIEW

**Review by satmandu — APPROVED** [review]
This is an insane amount of work!

We should probably get a script written to automate some of this if we can...

**Comment by Zopolis4** [comment]
Yeah, that's part of the reason why I'm doing this-- the simple packages that I've converted to use buildsystems and git version tags should be pretty easy to automate.

File: packages/libspiro.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  homepage 'https://github.com/fontforge/libspiro'
+  version '20221101'
+  license 'GPL3+'
+  compatibility 'all'
+  source_url 'https://github.com/fontforge/libspiro.git'
+  git_hashtag version
+  binary_compression 'tar.zst'
+
+  binary_sha256({
+    aarch64: 'aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa',
```
Comment (uberhacker) [inline packages/libspiro.rb:14]: Missing binary hash?

**Comment by uberhacker** [comment]
Forgot to add new packages and update renamed packages in `tools/packages.yaml`.

**Comment by uberhacker** [comment]
I appreciate all the effort you put into these updates @Zopolis4!  I was feeling alone for a long time.  In the future, it probably makes sense to split up into multiple PRs to make the process more manageable and easier to review.

**Comment by Zopolis4** [comment]
I'm trying to find a balance between a new PR for every package, because then I spend more time opening branches and writing PRs than doing actual work, but I agree this was probably too many in one go.

File: packages/libspiro.rb (hunk 1 above)
Comment (Zopolis4) [reply packages/libspiro.rb:14]: I have no idea how that even happened, given that `crew upload` should replace both `aarch64` and `armv7l` given that they have the same base hash.

Comment (uberhacker) [reply packages/libspiro.rb:14]: The process probably shouldn't rely on a template but rather generate the entire section if it isn't present.

**Comment by Zopolis4** [comment]
New and renamed packages are covered by `version.rb`.
image

**Pull request merged**